erc20_moc/Erc20.noblock.js

```diff
@@ -107,18 +105,9 @@ class ERC20 {
       await updateBalanceOfUser(
         dbClient,
         receiver,
-        this.symbol,
-        trx_time,
-        amount
-      );
-
-      await insertTransaction(
```

Comment:
are we supposed to insert a new record in transactions table each time we make an internal transaction, because we're not doing it in trading-engine atm?